test/components/isEvolved.ts

```diff
@@ -0,0 +1,56 @@
+import { IPokemon } from "../../src/components/Battles";
+import { IPokemonEvolution, IPokemonEvolutionByLevel } from "../../src/components/constants/Pokemon";
+import { FullScreenPokemon } from "../../src/FullScreenPokemon";
+import { it } from "../main";
+import { stubBlankGame } from "../utils/fakes";
+
+it("properly evolves a Pokemon at exactly its level requirement", (): void => {
+    // Arrange
+    const fsp: FullScreenPokemon = stubBlankGame();
+    const pokemonTitle: string[] = "CHARMANDER".split("");
```

Comment:
Thinking on this further - I think you can get rid of the requirement for `"CHARMANDER"` to exist by creating a new `IPokemon` instead of retrieving it from `constants`. Can you try that out? We'd talked about removing assumptions; this should fix that.